tools/backup/firebase.ts

```diff
@@ -41,13 +41,19 @@ const downloadGcpFile = async (file: File): Promise<BackupFile> => {
 	const writeStream = fs.createWriteStream(localPath);
 	return new Promise((resolve, reject) => {
 		readStream
-			.on('end', () =>
-				resolve({
+			.on('end', () => {
+				let size;
+				if (typeof metadata.size === 'string') {
+					size = parseInt(metadata.size, 10);
+				} else {
+					size = metadata.size;
+				}
+				return resolve({
 					localPath,
 					remotePath: file.name,
-					size: parseInt(metadata.size, 10),
-				}),
-			)
```

Comment:
`metadata.size` went from `string` to `string | number | undefined`.  I don't really know what IRL situation those types correspond to and couldn't find documentation but I think this should be fine